docs/src/pages/components/timeline/OppositeContentTimeline.tsx

```diff
@@ -12,19 +12,19 @@ export default function OppositeContentTimeline() {
   const timelineItems = [
     {
       activity: 'Eat',
-      time: '09:30AM',
+      time: '09:30 am',
     },
     {
       activity: 'Code',
-      time: '10:00AM',
+      time: '10:00 am',
     },
     {
       activity: 'Sleep',
-      time: '00:00AM',
+      time: '00:00 am',
```

Comment:
```suggestion
      time: '12:00 am',
```

(00:00 is for the 24hr clock...)